### ISSUE TITLE
Page 1: Abstract: Results

### ISSUE THREAD

**Issue by LTLA** [davismcc/scaterPaperExtras#1]
> The package provides a convenient, flexible workflow to go from raw reads through quality control to a tidy dataset ready for downstream analysis.

I would suggest rewording this to:

> The package provides a convenient, flexible workflow to process raw sequencing reads into a high-quality expression dataset ready for downstream analysis.

This is more formal than "go from" and avoids the difficulty of defining "tidy".

**Comment by davismcc** [comment]
Good change - applied.
